src/share/classes/com/sun/javatest/regtest/tool/Tool.java

```diff
@@ -459,6 +459,22 @@ public void process(String opt, String arg) throws BadArgs {
             }
         },
 
+        new Option(GNU, AGENT_POOL, null, "--agent-attempts") {
```

Comment:
It might be good to add a FAQ entry for this option as well.

Q. `jtreg` has trouble starting agents; what can I do?
A. This can happen because _<fill-in-the-blank>_. Use the `--sgents-attempts` option to retry creating a agent up to a given number of times.

Reply:
Hello Jon, I've now updated the PR to include a FAQ entry too.